### ISSUE TITLE
attribute 'company' missing form the user entity

### ISSUE THREAD

**Issue by belaie** [AzureADSamples/ConsoleApp-GraphAPI-DotNet#18]
Hello

I could not find the attribute 'company' on the user entity reference, i wonder why is this not available on the user entity from the azure graph api?

**Comment by dkershaw10** [comment]
We do sync this property from on premises, but it's not exposed today on the user entity.  Is this something you need?

**Comment by belaie** [comment]
Yes, we need company property to be synced. actually all the possible properties in AzureAD must be exposed.
